### PR TITLE
Crystopia Server v3

### PR DESCRIPTION
**Pull request by tnsjesper**
## Type of change

- [x ] Add new directory for a server
- [x ] Update directory of a server
- [x ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/crystopia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "play.crystopia.net"
+    ],
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
+      "web_support": "https://labymod.net/wiki",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:15]: Either add your support (FAQ or contact form) or remove key

File: minecraft_servers/crystopia/manifest.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      "twitter": "crystopianet",
+      "discord": "https://dc.crystopia.net",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:17]: Custom links are reserved for partners; use https://discord.gg/[INVITE]

File: minecraft_servers/crystopia/manifest.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "server_wildcards": [
+      "play.crystopia.net"
+    ],
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:14]: Not reachable

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/crystopia/manifest.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "server_wildcards": [
+      "play.crystopia.net"
```
Comment (maaaathis) [inline minecraft_servers/crystopia/manifest.json:6]: Doesn't look like a real wildcard

File: minecraft_servers/crystopia/manifest.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      "web_support": "https://labymod.net/wiki",
+      "twitter": "crystopianet",
+      "discord": "https://dc.crystopia.net",
+      "youtube": "https://www.youtube.com/@crystopia"
+    },
+    "gamemodes": {
+      "smp": {
+        "name": "SMP", 
+        "color": "#FFA500",
+        "url": "https://www.crystopia.net/",
```
Comment (maaaathis) [inline minecraft_servers/crystopia/manifest.json:24]: Just out of interest: What is the specific context between this (general) website for this game mode?

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/crystopia/manifest.json (hunk 5 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:24]: We don't have a separate page for game mode yet.

File: minecraft_servers/crystopia/manifest.json (hunk 4 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:6]: You can join under play.crystopia.net bur we have nicht t a real whitecard

File: minecraft_servers/crystopia/manifest.json (hunk 2 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:17]: I didn't know that, I'll fix it today

Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:17]: I didn't know that, I'll fix it today

File: minecraft_servers/crystopia/manifest.json (hunk 3 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:11]: Oh yes sorry i fix the site soon... Sorry I hope that is not bad)

**Comment by tnsjesper** [comment]
No i have all problems fixed without the shop site this is the site...

File: minecraft_servers/crystopia/manifest.json (hunk 5 above)
Comment (maaaathis) [reply minecraft_servers/crystopia/manifest.json]: Then remove?

File: minecraft_servers/crystopia/manifest.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      "web_support": "https://www.crystopia.net/wiki",
+      "twitter": "crystopianet",
+      "discord": "https://discord.com/invite/q7xW3MH425",
+      "youtube": "https://www.youtube.com/@crystopia"
+    },
+    "gamemodes": {
+      "smp": {
+        "name": "SMP", 
+        "color": "#FFA500",
+        "url": "https://www.crystopia.net/games/smp",
```
Comment (maaaathis) [inline minecraft_servers/crystopia/manifest.json:21]: Returns a 404

File: minecraft_servers/crystopia/manifest.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://www.crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
+      "web_support": "https://www.crystopia.net/wiki",
```
Comment (maaaathis) [inline minecraft_servers/crystopia/manifest.json:12]: It's literally the same content as the landing/home page. I think you mean https://www.crystopia.net/index.php?route=/wiki/ instead.

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/crystopia/manifest.json (hunk 7 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:12]: Yes my bad sorry so much

File: minecraft_servers/crystopia/manifest.json (hunk 6 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:21]: Yes i fix it later page Display some objekts not correct because its disabeled!

**Comment by tnsjesper** [comment]
I'm not allowed to set this...

**Comment by maaaathis** [comment]
> I'm not allowed to set this...
> 
> https://www.crystopia.net/index.php?route=/wiki/

`https://www.crystopia.net/index.php?route=/wiki` also works.
But I think you should double check your webserver configuration. As far as I know, there are options (for example via a .htaccess or nginx configuration) to avoid these in-dynamic routes to the real route of your choice.

**Comment by tnsjesper** [comment]
> > I'm not allowed to set this...
> > https://www.crystopia.net/index.php?route=/wiki/
> 
> `https://www.crystopia.net/index.php?route=/wiki` also works. But I think you should double check your webserver configuration. As far as I know, there are options (for example via a .htaccess or nginx configuration) to avoid these in-dynamic routes to the real route of your choice.

Yes i have fix it

**Comment by DoJapHD** [comment]
Server is not released yet; feel free to reopen the PR once v3 is available

**Comment by tnsjesper** [comment]
Ok i do it!

**Pull request closed without merging**